src/components/calculator.jsx

```diff
@@ -0,0 +1,40 @@
+function Calculator() {
+  return (
+    <div className="h-[100vh] flex justify-center items-center">
+      <ul className="bg-[#e0e0e0] w-4/5 grid columns-6 border-2 border-black/40">
+        <span className="p-2 text-right justify-self-end bg-[#858694] w-full text-white hover:bg-[#858694]">0</span>
+        <ul className="flex justify-center items-center">
+          <li className="flex justify-center border-2 border-black/40 w-1/4">AC</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">+/-</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">%</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">÷</li>
+        </ul>
+        <ul className="flex justify-center items-center">
+          <li className="flex justify-center border-2 border-black/40 w-1/4">7</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">8</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">9</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">x</li>
+        </ul>
+        <ul className="flex justify-center items-center">
+          <li className="flex justify-center border-2 border-black/40 w-1/4">4</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">5</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">6</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">-</li>
+        </ul>
+        <ul className="flex justify-center items-center">
+          <li className="flex justify-center border-2 border-black/40 w-1/4">1</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">2</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">3</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">+</li>
+        </ul>
+        <ul className="flex justify-center items-center">
+          <li className="flex justify-center border-2 border-black/40 w-1/2">0</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">.</li>
+          <li className="flex justify-center border-2 border-black/40 w-1/4">=</li>
+        </ul>
+      </ul>
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
- [ ] As part of the requirements all components should be [Class-Based](https://www.w3schools.com/react/react_class.asp) components,  kindly refactor your code, to be specific the [calculator](https://github.com/ohyajk/Math-Magicians/pull/2/files#diff-463535bbb18a05ee401cee0ab5d354fbad047ad29cdf615a2fb0d76fd71bd607) & the [App.js](https://github.com/ohyajk/Math-Magicians/pull/2/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67) files to be class component instead of functional ones  🙏 

![image](https://user-images.githubusercontent.com/89282221/211313485-b655f95a-c29c-43f9-b1ce-a22b5f15e573.png)

Reply:
Kindly ask for another review from your dashboard as soon as you are ready 🙏

Reply:
Yes, I did and It got approved. Thank you once again.❤️